src/Microsoft.VisualStudio.Threading.Analyzers/Strings.resx

```diff
@@ -309,4 +309,10 @@ Use AsyncLazy&lt;T&gt; instead.</value>
     <value>Call ThrowIfCancellationRequested()</value>
     <comment>"ThrowIfCancellationRequested" is a method name and should not be translated.</comment>
   </data>
-</root>
+  <data name="VSTHRD112_MessageFormat" xml:space="preserve">
+    <value>Avoid returning null from a non-async awaitable method</value>
```

Comment:
The problem is very specifically `Task` or `Task<T>` returning methods as those are the ones callers should never expect a null result from. Other return types are allowed to return null and/or default.
Can you change this to "Avoid returning null from a `Task`-returning method."?